ffs/integrationtest/integration_test.go

```diff
@@ -454,12 +461,14 @@ func TestUnfreeze(t *testing.T) {
 }
 
 func TestRenew(t *testing.T) {
+	// ToDo: unskip when testnet/3 sectormock allows more than one deal
+	// See https://github.com/filecoin-project/lotus/issues/1455
+	t.SkipNow()
```

Comment:
The only test that was _skipped_ in the testnet/3 migration. A bit annoying, hope fix soon.